model-optimizer/mo/utils/guess_framework.py

```diff
@@ -42,3 +42,5 @@ def guess_framework_by_ext(input_model_path: str) -> int:
         return 'kaldi'
     elif re.match(r'^.*\.onnx$', input_model_path):
         return 'onnx'
+    elif re.match(r'^.*\.pdmodel$', input_model_path):
```

Comment:
Technically Paddle  model can be represented by 'folder', not a single file. This folder will contain separate bin file for different weights

Reply:
Removed.